### PR TITLE
Contributor profiles

### PR DESCRIPTION
**Pull request by skddc**
This adds the basic layout and functionality for rendering (linkable) details in a third column/pane, and as first addition it implements basic contributor profile cards. Account links and stats are still missing, but easy to add from here on.

See commits for some more details.

Screenshot from 2019-07-11 10-05-44

### CODE REVIEW

**Comment by skddc** [comment]
Now also includes account links:

Screenshot from 2019-07-11 15-31-17

File: app/components/external-account-link/component.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  tagName: "",
+  account: null,
+
+  iconComponentName: computed('account.site', function() {
+    if (this.account.site.match(/github|gitea|wiki/)) {
+      console.log('wtf');
```
Comment (galfert) [inline app/components/external-account-link/component.js]: Necessary information? ;)

Comment (skddc) [reply app/components/external-account-link/component.js]: Haha, good catch. :)

```suggestion
```

File: app/templates/dashboard/contributors/show.hbs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        </li>
+      {{/if}}
+    </ul>
+  </div>
+
+  <div class="actions">
+    <p>
+      <a href="https://rinkeby.etherscan.io/address/{{model.account}}" class="button small" target="_blank" rel="noopener">Inspect Ethereum transactions</a>
+      {{#if model.ipfsHash}}
+        <a href="https://ipfs.io/ipfs/{{model.ipfsHash}}" class="button small" target="_blank" rel="noopener">Inspect IPFS profile</a>
```
Comment (bumi) [inline app/templates/dashboard/contributors/show.hbs]: we could use the `gatewayUrl` here

Comment (skddc) [reply app/templates/dashboard/contributors/show.hbs]: That was actually intentional, for two reasons:

1.) Have it be an independent source
2.) Have stuff be cached on ipfs.io so more nodes can distribute it

**Pull request merged**